package.json

```diff
@@ -17,8 +24,32 @@
   "author": "Procech",
   "license": "MIT",
   "dependencies": {
+    "@babel/polyfill": "^7.12.1",
+    "bcryptjs": "^2.4.3",
+    "body-parser": "^1.19.0",
     "colors": "^1.4.0",
+    "cors": "^2.8.5",
+    "crossenv": "0.0.2-security",
+    "dotenv": "^8.2.0",
+    "eslint-config-airbnb-base": "^14.2.1",
+    "eslint-config-prettier": "^8.0.0",
+    "express": "^4.17.1",
+    "fs": "0.0.1-security",
+    "joi": "^17.4.0",
+    "jsonwebtoken": "^8.5.1",
     "morgan": "^1.10.0",
+    "mysql2": "^2.2.5",
+    "sequelize": "^6.5.0",
+    "sequelize-cli": "^6.2.0"
+  },
+  "devDependencies": {
+    "@babel/cli": "^7.12.17",
+    "@babel/core": "^7.12.17",
+    "@babel/node": "^7.12.17",
+    "@babel/preset-env": "^7.12.17",
+    "eslint": "^5.16.0",
+    "eslint-plugin-import": "^2.22.1",
+    "jshint": "^2.12.0",
```

Comment:
Please if you are not using a package you need to remove it before we merge.

Reply:
As I told you earlier, these are necessary but am just not using them now but am using them later